test/project-service.ts

```diff
@@ -153,7 +154,7 @@ describe("Project Service Tests", () => {
 				"readme": "dummy",
 				"repository": "dummy"
 			});
-			npmInstallationManager.install("tns-template-hello-world", defaultTemplateDir, {dependencyType: "save"}).wait();
+			npmInstallationManager.install("tns-template-hello-world", defaultTemplateDir, { dependencyType: "save" }).wait();
```

Comment:
can use `constants.RESERVED_TEMPLATE_NAMES["default"]` instead of `"tns-template-hello-world"`